### PR TITLE
Add ability to invert injections in minifollowups

### PR DESCRIPTION
**Pull request by spxiwh**
Here's the change needed to add inverted injections into the minifollowups. I do this using the already-existing "inj-scale-factor" argument to inject.py, which, if set to -1, injects the signal inverted in the time domain.

(This needs also ini file change)

### CODE REVIEW

File: pycbc/strain.py [hunk 1]
```diff
@@ -349,6 +353,10 @@ def insert_strain_option_group(parser, gps_times=True):
                       help="(optional) Injection file used to add "
                       "sine-Gaussian burst waveforms into the strain")
 
+    data_reading_group.add_argument("--injection-scale-factor", type=float,
+                    default=1, help="Multiple injections by this factor "
+                    "before injecting into the data.")
```
Comment (ahnitz) [inline pycbc/strain.py]: This pedantic, but this should by "multiply", no?

**Comment by ahnitz** [comment]
Ian, thanks for splitting this apart. If I read this right, then this is a fairly straightforward solution. It does however make the plot non-optional, so I think it would be good to have the code change to the ini file in hand at the same time. Otherwise, I don't have any issues with this patch, save a small typo.

**Comment by spxiwh** [comment]
I would change Multiple -> Multiply, except it actually divides by this factor, so I made it say that instead!

I'll prepare an ini file change to accompany this.

**Comment by spxiwh** [comment]
See https://code.pycbc.phy.syr.edu/ligo-cbc/pycbc-config/merge_requests/112 for ini file change

**Pull request merged**